qiskit/transpiler/passes/layout/apply_layout.py

```diff
@@ -83,9 +83,11 @@ def run(self, dag):
             full_layout = Layout()
             old_phys_to_virtual = layout.get_physical_bits()
             new_virtual_to_physical = post_layout.get_virtual_bits()
+            phys_map = {i: i for i in range(len(new_dag.qubits))}
```

Comment:
Super minor, but isn't this just a list?

Reply:
Heh, yeah it is, I'll refactor it in a bit. That's what I get for writing code at the end of a long release day. :)

Reply:
Done in: https://github.com/Qiskit/qiskit-terra/pull/10466/commits/a26fa48ae0058438746c3ff0c49887d903128de1